src/index.js

```diff
@@ -11,7 +11,7 @@ const debug = require('debug')('start-server-and-test')
 /**
  * Used for timeout (ms)
  */
-const fiveMinutes = 5 * 60 * 1000
+const waitOnTimeout = process.env.WAIT_ON_TIMEOUT || 5 * 60 * 1000
```

Comment:
I think we both need good default name and convert from env string to a number
```suggestion
const fiveMinutes = 5 * 60 * 1000
const waitOnTimeout = 
  process.env.WAIT_ON_TIMEOUT ?
  Number(process.env.WAIT_ON_TIMEOUT) : 5 * 60 * 1000
```